contracts/src/v0.8/automation/dev/test/AutomationRegistry2_3.t.sol

```diff
@@ -9,13 +9,15 @@ import {AutomationRegistryLogicB2_3} from "../v2_3/AutomationRegistryLogicB2_3.s
 import {IAutomationRegistryMaster2_3, AutomationRegistryBase2_3} from "../interfaces/v2_3/IAutomationRegistryMaster2_3.sol";
 import {ChainModuleBase} from "../../chains/ChainModuleBase.sol";
 import {MockV3Aggregator} from "../../../tests/MockV3Aggregator.sol";
+import {ERC20Mock} from "../../../vendor/openzeppelin-solidity/v4.8.3/contracts/mocks/ERC20Mock.sol";
 
 contract AutomationRegistry2_3_SetUp is BaseTest {
   address internal LINK_USD_FEED;
   address internal NATIVE_USD_FEED;
   address internal FAST_GAS_FEED;
-  address internal constant LINK_TOKEN = 0x1111111111111111111111111111111111111113;
+  address internal constant FINANCE_ADMIN_ADDRESS = 0x1111111111111111111111111111111111111114;
   address internal constant ZERO_ADDRESS = address(0);
+  address internal constant ADMIN = address(uint160(uint256(keccak256("ADMIN"))));
```

Comment:
we can do this in a follow up, but could we rename to `UPKEEP_ADMIN` to differentiate from finance admin?